group/views.py

```diff
@@ -22,25 +22,23 @@
 SOFTWARE.'''
 from django.http import HttpResponse
 from django.shortcuts import render_to_response, render
+from group.models import Group
+import datetime
+
+def get_running_contest(request, group_id):
+        
+        group = Group.objects.get(id = group_id)
```

Comment:
Please check whether the group id valid or not. (Does the group object exist?)
If not, write a warning log.